### PR TITLE
[5.8] Enable JS on StyleCI

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

**Comment by driesvints** [comment]
We've decided against this for now for exactly the change on the webpack.mix.js file.

**Comment by GrahamCampbell** [comment]
Let me see if we can avoid that.

**Comment by driesvints** [comment]
@GrahamCampbell can you maybe do this on a fork? (see all the auto-fixes by StyleCI) 

Notification overload for me atm 😅

**Comment by GrahamCampbell** [comment]
It's ready now. :)

File: .styleci.yml [hunk 1]
```diff
@@ -6,4 +6,9 @@ php:
     not-name:
       - index.php
       - server.php
+js:
+  print-width: 100
```
Comment (driesvints) [inline .styleci.yml]: Let's remove this one to stick to the default like we do for all other repos.

**Comment by driesvints** [comment]
Can you also squash your commits?

File: .styleci.yml (hunk 1 above)
Comment (GrahamCampbell) [reply .styleci.yml]: If we leave it at 120, 3 other files will be changed I think?

Comment (driesvints) [reply .styleci.yml]: I don't think so? Wasn't the case when I enabled it.

**Comment by taylorotwell** [comment]
Squashed and merged.

**Comment by GrahamCampbell** [comment]
🍻

**Pull request merged**